src/script/common/c_content.cpp

```diff
@@ -1851,7 +1851,10 @@ void read_hud_element(lua_State *L, HudElement *elem)
 	elem->name    = getstringfield_default(L, 2, "name", "");
 	elem->text    = getstringfield_default(L, 2, "text", "");
 	elem->number  = getintfield_default(L, 2, "number", 0);
-	elem->item    = getintfield_default(L, 2, "item", 0);
+	if (elem->type == HUD_ELEM_WAYPOINT)
```

Comment:
should add a comment here noting that waypoints reuse `item`

Reply:
will do

Reply:
This doesn't really explain it, I was thinking more of:
`// Waypoints reuse the item property to store the precision (+1) of the displayed distance`